lesson_02/utils.js

```diff
@@ -0,0 +1,10 @@
+export const handleError = (res, e) => {
+    if (e.isJoi) {
+        res.status(422).json({ message: e.message });
```

Comment:
in case of joi error, let's return 400 status code

Reply:
Done